diff --git a/lib/generators/jinda/install_generator.rb b/lib/generators/jinda/install_generator.rb
index e79330d..4515039 100755
--- a/lib/generators/jinda/install_generator.rb
+++ b/lib/generators/jinda/install_generator.rb
@@ -1,7 +1,6 @@
 module Jinda
   module Generators
     class InstallGenerator < Rails::Generators::Base
-      require 'pry'
       desc "Install jinda component to existing Rails app "
       def self.source_root
         File.dirname(__FILE__) + "/templates"
diff --git a/lib/tasks/jinda.rake b/lib/tasks/jinda.rake
index 1f3c5f4..ce64112 100755
--- a/lib/tasks/jinda.rake
+++ b/lib/tasks/jinda.rake
@@ -16,11 +16,27 @@ namespace :jinda do
 
   desc "generate admin user"
   task :seed=> :environment do
-    unless Identity.where(code:"admin").exists?
-      identity= Identity.create :code => "admin", :email => "admin@test.com", :password => "secret",
-				:password_confirmation => "secret", :image => "https://user-images.githubusercontent.com/3953832/42472827-50ed8cbc-8388-11e8-8982-fa523c25288f.png"
-      User.create :provider => "identity", :uid => identity.id.to_s, :code => identity.code,
-				:email => identity.email, :role => "M,A,D", :auth_token => "71JxMH5fxi23zinBoq1uKA", :image => identity.image
+    usamples  = Array.new
+    ucreated  = Array.new
+    uexist    = Array.new
+    usamples  = [
+      { code: 'admin', password: 'secret', email: 'admin@test.com', role: 'M,A,D'},
+      { code: 'tester', password: 'password', email: 'tester@test.com', role: 'M'}
+    ]
+    usamples.each do |h|
+      code  = h[:code].to_s
+      email = h[:email].to_s
+      password = h[:password].to_s
+      role = h[:role].to_s
+      unless Identity.where(code: code).exists?
+        identity= Identity.create :code => code, :email => email, :password => password , :password_confirmation => password
+        User.create :provider => "identity", :uid => identity.id.to_s, :code => identity.code,:email => identity.email, :role => role
+        ucreated < h
+      end
+    end
+    uexist = usamples - ucreated
+    uexist.each do |e|
+      puts "#{e[:code].to_s} with password = #{e[:password].to_s} already exist"
     end
   end
 
